components/tooltip/tooltip.js

```diff
@@ -772,6 +778,7 @@ class Tooltip extends RtlMixin(LitElement) {
 			this.dispatchEvent(new CustomEvent(
 				'd2l-tooltip-show', { bubbles: true, composed: true }
 			));
+			if (this.announced && !this._isInteractive(this._findTarget())) announce(this.innerText);
```

Comment:
One concern here is that if `!this._isInteractive(this.__target)` returns `false` it doesn't guarantee the tooltip won't be announced, all it guarantees is that it doesn't announce properly on all browser / screen reader combinations which could cause some duplicate announcements.

Thoughts on removing `aria-describedby` if `this.announced && !this._isInteractive(this.__target)` is `true`?

Reply:
Yup that seems like a good idea.